Makefile

```diff
@@ -3,6 +3,7 @@ dbt-trino-tests:
 	./docker/dbt/build.sh
 	./docker/init_trino.bash
 	pip install -r dev_requirements.txt
+	./docker/run_tests.bash
```

Comment:
Leave the `Makefile` out of this PR because fixing it is not really in the scope.

Reply:
ok. Will revert this.

Reply:
oh I will add them in separate PR, thanks for catching this